src/Action/SetObjectFieldValueAction.php

```diff
@@ -112,15 +142,31 @@ public function __construct(
      */
     public function __invoke(Request $request): JsonResponse
     {
+        // NEXT_MAJOR: Remove this BC-layer.
+        if (null === $request->get('_sonata_admin')) {
+            @trigger_error(
+                'Not passing "_sonata_admin" value in the request is deprecated since sonata-project/admin-bundle 3.x'
```

Comment:
```suggestion
                'Not passing "_sonata_admin" value in the request is deprecated since sonata-project/admin-bundle 3.x'
```
```suggestion
                'Not passing "_sonata_admin" parameter in the request is deprecated since sonata-project/admin-bundle 3.x'
```